ui/tests/acceptance/secrets/backend/azure/azure-configuration-test.js

```diff
@@ -0,0 +1,144 @@
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { overrideResponse } from 'vault/tests/helpers/stubs';
+import { SECRET_ENGINE_SELECTORS as SES } from 'vault/tests/helpers/secret-engine/secret-engine-selectors';
+import { mountBackend } from 'vault/tests/helpers/components/mount-backend-form-helpers';
+import {
+  expectedConfigKeys,
+  expectedValueOfConfigKeys,
+  configUrl,
+} from 'vault/tests/helpers/secret-engine/secret-engine-helpers';
+
+module('Acceptance | Azure | configuration', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function () {
+    const flash = this.owner.lookup('service:flash-messages');
+    this.store = this.owner.lookup('service:store');
+    this.flashSuccessSpy = spy(flash, 'success');
+    this.flashInfoSpy = spy(flash, 'info');
+    this.version = this.owner.lookup('service:version');
+    this.uid = uuidv4();
+    return authPage.login();
+  });
+  module('isEnterprise', function (hooks) {
+    hooks.beforeEach(function () {
+      this.version.type = 'enterprise';
+    });
+
+    test('it should show empty state and navigate to configuration view after mounting the azure engine', async function (assert) {
+      const path = `azure-${this.uid}`;
+      await visit('/vault/settings/mount-secret-backend');
+      await mountBackend('azure', path);
+
+      assert.dom(GENERAL.emptyStateTitle).hasText('Azure not configured');
+      assert.dom(GENERAL.emptyStateActions).doesNotContainText('Configure Azure');
+      // cleanup
+      await runCmd(`delete sys/mounts/${path}`);
+    });
+
+    test('it should not show "Configure" from toolbar', async function (assert) {
+      const path = `azure-${this.uid}`;
+      await enablePage.enable('azure', path);
+      assert.dom(SES.configure).doesNotExist('Configure button does not exist.');
+      assert.strictEqual(
+        currentURL(),
+        `/vault/secrets/${path}/configuration`,
+        'navigated to configuration view'
+      );
+      // cleanup
+      await runCmd(`delete sys/mounts/${path}`);
+    });
+
+    test('it should show configuration with WIF options configured', async function (assert) {
+      const path = `azure-${this.uid}`;
+      const type = 'azure';
+      const wifAttrs = {
+        subscription_id: 'subscription-id',
+        tenant_id: 'tenant-id',
+        client_id: 'client-id',
+        identity_token_audience: 'audience',
+        identity_token_ttl: 720000,
+        environment: 'AZUREPUBLICCLOUD',
+      };
+      this.server.get(`${path}/config`, () => {
+        assert.ok(true, 'request made to config when navigating to the configuration page.');
+        return { data: { id: path, type, ...wifAttrs } };
+      });
+      await enablePage.enable(type, path);
+      for (const key of expectedConfigKeys('azure-wif')) {
+        assert.dom(GENERAL.infoRowLabel(key)).exists(`${key} on the ${type} config details exists.`);
```

Comment:
nit: since we're checking the value below which relies on the `key` we don't need this assertion as well